src/engraving/rw/read410/tread.cpp

```diff
@@ -3436,6 +3436,10 @@ void TRead::read(MMRest* r, XmlReader& e, ReadContext& ctx)
             NoteDot* dot = Factory::createNoteDot(r);
             TRead::read(dot, e, ctx);
             r->add(dot);
+        } else if (tag == "mmRestNumberPos") {
+            // Old property, deprecated in 4.5
+            r->setNumberOffset(e.readDouble() - ctx.style().styleS(Sid::mmRestNumberPos).val());
+        } else if (tag == "mmRestNumberPos") {
```

Comment:
This one looks unneeded - did you mean
`else if (TRead::readProperty(r, tag, e, ctx, Pid::MMREST_NUMBER_OFFSET))`?